package.json

```diff
@@ -131,6 +131,9 @@
     "yarn": "3.5.0"
   },
   "resolutions": {
-    "@babel/traverse": "7.22.5"
+    "@babel/traverse": "7.22.5",
+    "vue": "2.7.14",
```

Comment:
Out of interest what was the reason for the vue resolutions?

Reply:
@siggerzz is helping with this 🎉 trying to fix the failing nuxt builds for the example apps 🤞